src/Message/ServerRestRefundRequest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Omnipay\SagePay\Message;
+
+use Omnipay\SagePay\Message\ServerRestRefundResponse;
+
+/**
+ * Sage Pay REST Server Refund Request
+ */
+class ServerRestRefundRequest extends AbstractRestRequest
+{
+    public function getService()
+    {
+        return static::SERVICE_REST_TRANSACTIONS;
+    }
+
+    /**
+     * @return string the transaction type
+     */
+    public function getTxType()
+    {
+        return ucfirst(strtolower(static::TXTYPE_REFUND));
+    }
+
+    /**
+     * Add the optional token details to the base data.
+     *
+     * @return array
+     */
+    public function getData()
+    {
+        $data = $this->getBaseData();
+
+        $data['transactionType'] = $this->getTxType();
+        $data['vendorTxCode'] = $this->getTransactionId();
+        $data['description'] = $this->getDescription();
+        $data['amount'] = (int) $this->getAmount();
```

Comment:
As with the purchase request, I think this should be...
```suggestion
        $data['amount'] = $this->getAmountInteger();
```